bower.json

```diff
@@ -3,9 +3,9 @@
   "dependencies": {
     "ember": "~2.8.0",
     "ember-cli-shims": "0.1.1",
-    "Faker": "3.1.0",
     "perfect-scrollbar": "~0.6.10",
-    "pretender": "~0.12.0"
-  },
-  "resolutions": {}
-}
\ No newline at end of file
+    "mocha": "~2.2.4",
+    "chai": "~2.3.0",
+    "ember-mocha-adapter": "~0.3.1"
```

Comment:
mocha, chai and ember-mocha-adapter can be removed once this `"ember-cli-mocha": "~0.13.2",` is set in the `package.json`